tests/functional/aws-node-sdk/test/multipleBackend/get.js

```diff
@@ -5,6 +5,7 @@ import BucketUtility from '../../lib/utility/bucket-util';
 const bucket = 'buckettestmultiplebackendget';
 const memObject = 'memobject';
 const fileObject = 'fileobject';
+const awsObject = 'awsobject';
 const emptyObject = 'emptyObject';
 const body = Buffer.from('I am a body', 'utf8');
```

Comment:
we should also test putting a larger object.  you can create a 1GB buffer and put it.